gmime/part.go

```diff
@@ -103,6 +103,14 @@ func (p *Part) Headers() textproto.MIMEHeader {
 	return nil
 }
 
+// GetContentID returns the content ID (if there is one) per mime part
+// we expect all attachments to have a content-id
+func (p *Part) GetContentID() string {
+	cCID := C.g_mime_object_get_content_id(p.gmimePart)
+	defer C.g_free(C.gpointer(unsafe.Pointer(cCID)))
```

Comment:
What happens guessing it returns empty string if there is not one? not too familiar with `C.g_free` stmt if we get null pointer back

Reply:
so looking at other spots in the code for null checks I fund this guy https://github.com/sendgrid/go-gmime/blob/539930aaa3d4ffe8abc1b882e93671c63a9ddf7c/gmime/part.go#L64
which calls this guy https://github.com/sendgrid/go-gmime/blob/539930aaa3d4ffe8abc1b882e93671c63a9ddf7c/gmime/gmime.c#L17
which can return null but the first guy doesn't do null check and defers the free stmt...

resolving comment